src/context/AppProvider/AppContext.tsx

```diff
@@ -100,6 +100,8 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
 
   // Initial account and portfolio fetch for non-ledger wallets
   useEffect(() => {
+    dispatch(portfolio.actions.setIsAccountLoading(true))
```

Comment:
q: can `getAccount` handle this internally?

Reply:
Unfortunately it looks like it's starting way too late by doing this and so the UI is displaying first that the wallet doesn't support this chain before the loading state

Reply:
Discussed this one out, agreed to keep the current implementation for now